juniper/src/util.rs

```diff
@@ -51,3 +51,22 @@ fn test_to_camel_case() {
     assert_eq!(&to_camel_case("a")[..], "a");
     assert_eq!(&to_camel_case("")[..], "");
 }
+
+#[cfg(feature = "tracing-futures")]
```

Comment:
@ArsileLuci why `"tracing-futures"`? It will complicate looking for feature-gated stuff. It's better to gate everything under the single `"tracing"` feature, unless these two are going to be used separately, which is not our case.